server/manage.py

```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'learndjango.settings')
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings.settings')
```

Comment:
Why `config.settings.settings` ideally it should be `config.settings.local` / `config.settings.production`

Reply:
updated in the new commit as `config.settings.local` and settings file is renamed as `local.py`